tests/test_pipreqs.py

```diff
@@ -82,6 +83,14 @@ def setUpClass(cls):
         cls.requirements_path = os.path.join(cls.project, "requirements.txt")
         cls.alt_requirement_path = os.path.join(cls.project, "requirements2.txt")
         cls.non_existing_filepath = "xpto"
+        
+        cls.project_with_notebooks = os.path.join(os.path.dirname(__file__), "_data_notebook")
+        cls.project_with_invalid_notebooks = os.path.join(os.path.dirname(__file__), "_invalid_data_notebook")
+        cls.compatible_files = {
+            "original": os.path.join(os.path.dirname(__file__), "_data/test.py"),
+            "notebook": os.path.join(os.path.dirname(__file__), "_data_notebook/test.ipynb"),
+        }
+        cls.requirements_notebook_path = os.path.join(cls.project_with_notebooks, "requirements.txt")
```

Comment:
maybe rename this ine as well? to me it would make more sense as `notebook_requirement_path` or some other qualifier in place of notebook to better explicitate the associated test (if it is used in a single test, move this declaration to it?)